vars/unitTestPost.groovy

```diff
@@ -80,15 +86,9 @@ def call(Map config = [:]) {
   if (config['valgrind_stash']) {
     def valgrind_pattern = config.get('valgrind_pattern', '*.memcheck.xml')
     stash name: config['valgrind_stash'], includes: valgrind_pattern
-  } else {
-
-    // Need to leave this logic in here for backwards compatibility.
-    // Valgrind results need to stashed and reported in a common stage
-    // After all Valgrind tests are run.
-    valgrindReportPublish ignore_failure: ignore_failure,
-                          valgrind_stashes: []
   }
-  if (!stage_info['with_valgrind']) {
+  def record_issues = config.get('record_issues', true)
+  if ((!stage_info['with_valgrind'] && record_issues) || stage_info['NLT']) {
     cb_result = currentBuild.result
```

Comment:
This cb_result is global to the entire Jenkins job in progress.

Reply:
I've pushed a new version to make this local.